db/migrate/20180413111835_irreversible_migration_example.rb

```diff
@@ -0,0 +1,19 @@
+class IrreversibleMigrationExample < ActiveRecord::Migration[5.1]
+  # def up
+  #   remove_column :users, :email
+  # end
+
+  # def down
+  #   # It raises ActiveRecord::IrreversibleMigration exception
+  #   # To roll back this migration , we would have to add it
+  #   add_column :users, :email, :string
+  # end
+  def change
+    reversible do |dir|
+      change_table :products do |t|
+        dir.up   { t.change :price, :decimal, :precision => 8, :scale => 3 }
```

Comment:
please don't change the precision of price column. It may be used in other exercises